pages/portfolio.html

```diff
@@ -4,9 +4,51 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>✨Portfolio✨</title>
+  <link rel="stylesheet" href="../styles/style.css">
 </head>
+  <header>
+    <h1>K. Buslach </h1>
+    <h1>ADA C17 Sharks 🦈</h1>
+    <ul>
+      <nav>
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="/about.html">About me</a></li>
+        <li><a href="/portfolio.html">Portfolio</a></li>
```

Comment:
These relative paths will probably need to be updated if index.html is moved to the top level of the project